### ISSUE TITLE
why batch norm here?

### ISSUE THREAD

**Issue by budui** [menyifang/ADGAN#15]
Hi, thanks for your great work first!

I'm trying to reproduce your code, but I can not understand why these code using `F.batch_norm` in `AdaptiveInstanceNorm2d`, why not just `F.instance_norm`?

https://github.com/menyifang/ADGAN/blob/4dd70649ad136829b92dd6a1a823af7594a0220f/models/model_adgen.py#L355-L362

**Comment by budui** [comment]
Sorry for disturbing you. After resharp, batch_norm can do same thing with instance norm directly.
